docs/cse/ingestion/aws-cloudtrail.md

```diff
@@ -41,25 +34,10 @@ It’s also possible to configure individual sources to forward to CSE, as descr
 
 ### Configure an AWS CloudTrail Source
 
-1. In Sumo Logic, select **Manage Data > Collection > Collection**. 
-1. Navigate to the Hosted Collector where you want to create the source.
-1. On the **Collectors** page, click **Add Source** next to a Hosted Collector.
-1. Select **AWS CloudTrail**. 
-1. The page refreshes.  
-    ![cloudtrail-source.png](/img/cse/cloudtrail-source.png)
-1. **Name**. Enter a name for the source. 
-1. **Description**. (Optional) 
-1. **S3 Region**. Choose the AWS Region the S3 bucket resides in.
-1. **Bucket Name**. The name of your organizations S3 bucket as it appears in AWS.
-1. **Path Expression**. The path expression of the log file(s) in S3, can contain wildcards to include multiple log files.
-1. **Source Category**. Enter a string to tag the output collected from the source. The string that you supply will be saved in a metadata field called `_sourceCategory`.
-1. **Fields**.
+1. To configure a CloudTrail Source, see [Configure an AWS CloudTrail source](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudtrail-source/#configure-an-aws-cloudtrail-source).
+1. **Please configure Fields as shown below to forward cloudtrail logs to the CSE platform**.
     1. If you are not forwarding all sources in the hosted collector to CSE, click the **+Add Field** link, and add a field whose name is `_siemForward` and value is *true*. This will ensure all logs for this source are forwarded to CSE.
```

Comment:
```suggestion
    1. If you are not forwarding all sources in the hosted collector to CSE, click the **+Add Field** link, and add a field whose name is `_siemForward` and value is *true*. This will ensure all logs for this source are forwarded to Cloud SIEM.
```